playbook-website/app/javascript/site_styles/docs/_kit_doc.scss

```diff
@@ -43,6 +43,10 @@
     display: block;
   }
 
+  & > [class^=pb_card_body_kit] {
```

Comment:
This piece is kinda odd. Trying to remember why this was done.
@nickamantia @carloslimasd 

Any ideas on why we needed to tweak the site scss?

Reply:
@jasperfurniss Yeah that does seem odd. Ill take a look and see what happens when this is removed

Reply:
@jasperfurniss @carloslimasd I cant seem to figure out what this does. Ive commented out the code and i don't see any changes. The only thing i can think of is i put this here when the the rails card was automatically adding the body sub component and then never removed this when we fixed that issue.